ui/components/multichain/product-tour-popover/index.scss

```diff
@@ -9,6 +9,11 @@
     right: 6px !important;
   }
 
+  &__permissions-page-tour {
```

Comment:
@darkwing Sure I can leave a comment in the code, but basically I use !important here  to override a `left: -7px !important` that works fine for the base Product Tour (I think it's needed because of the productTourDirection prop), but if I try to use ProductTour component elsewhere, it applies a `left: -7px !important` which makes it uncentered.